mantidproject/mantid
# Indirect - QENS Fitting - Fix the output format of the QENS fitting algorithms

The output format of the QENS fitting algorithms is currently incorrect; the result parameters for each data-set are appended next to one another in a single workspace.

This PR modifies the existing QENS fitting routines, such that a workspace group is output with the parameters of each data-set.

**This PR must be merged after #22624**

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > ConvFit.
2. Click on the 'Multiple Input' tab button.
3. Use the 'Add Workspace' button to add the provided test data (irs26173 and irs26176, both with resolution irs26173). Any option may be used in the spectra input.
4. Select 'One Lorentzian' in the 'Fit Type' drop-down menu.
5. Check the check-box labelled 'Use Delta Function'.
6. Select 'Flat Background' in the 'Background Type' drop-down menu.
7. Click 'Run' and wait for the fit to complete.
8. Ensure a '*_Result' workspace group has been created and it contains a result for irs26173 and irs26176, both of which contain FWHM, Amplitude, Height and EISF parameter values.



Fixes #22672. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
